php_env/PHP_8.0/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM php:8.0.3-cli
+
+# Update and import specific required librairies
+
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y \
+	unzip \
+	libicu-dev \
+    libonig-dev \
+	gcc \
+	wget \
+	zlib1g-dev \
+    libzip-dev
+
+# Parametrize PHP
+
+RUN docker-php-ext-install mbstring
+RUN docker-php-ext-install intl
+RUN docker-php-ext-install zip
+RUN docker-php-ext-install pcntl
+RUN docker-php-source delete
+
+# Install composer
+COPY composer.sh /
+RUN chmod +x composer.sh
+RUN /composer.sh
+RUN mv composer.phar /usr/local/bin/composer
+RUN mkdir /.composer && chmod o+rwx /.composer
```

Comment:
You could use the alpine php image variant and install composer from `apk`

Or use 
```Dockerfile
# copy the Composer PHAR from the Composer image into the PHP image
COPY --from=composer /usr/bin/composer /usr/local/bin/composer
```

See: https://medium.com/@othillo/adding-composer-to-php-docker-images-using-multi-stage-builds-2a10967ae6c1

Reply:
> You could use the alpine php image variant and install composer from `apk`

Yes, it could be done also with debian apt, of course, but I like the composer.sh which ensures to be on the latest version of composer.

> Or use
> 
> ```dockerfile
> # copy the Composer PHAR from the Composer image into the PHP image
> COPY --from=composer /usr/bin/composer /usr/local/bin/composer
> ```

I really like this one, it seems to be the cleanest and it ensures the composer's version used. Thanks @williamdes

I will do it this way for any future project.